detail.css

```diff
@@ -0,0 +1,27 @@
+.smallpics {
+  width: 80px;
+  float: left;
+  display: flex;
+}
+
+.mainpic {
+  width: 430px;
+  float:left;
+}
+
+.mainpic img {
+  display: inline-block;
+  border: 1px solid #147871;
+  border-radius: 7px;
+  width: 350px;
+  margin: 0px 20px 0px 60px;
+  position: relative;
+}
+
+.smallpics img {
```

Comment:
The class could be moved to apply to each image instead of having the div hold the class and apply it to the child img tags.

Reply:
Thanks I learned about that after I completed this part!